linux_os/guide/system/accounts/accounts-session/file_permissions_home_dirs/bash/shared.sh

```diff
@@ -0,0 +1,12 @@
+# platform = multi_platform_all
+# reboot = false
+# strategy = restrict
+# complexity = low
+# disruption = low
+
+for home_dir in $(awk -F':' '{ if ($4 >= {{{ uid_min }}} && $4 != 65534) print $6 }' /etc/passwd); do
```

Comment:
What is that `65534`? A max uid? Would it be possible to make an explanatory variable declared in the macro definitions file?

Reply:
This uid is used in the special case of `nobody` user.